.bazelci/presubmit.yml

```diff
@@ -63,6 +67,11 @@ platforms:
     build_targets:
     - "--"
     - "..."
+    # These targets produce files and directories and it makes them fail to build on RBE.
```

Comment:
Lets hold off submitting this until we can fix this issue,

Reply:
Simple tests under tests/contrib now run on CI.